src/fheroes2/game/difficulty.h

```diff
@@ -40,7 +40,9 @@ namespace Difficulty
     std::string String( int );
 
     int GetScoutingBonus( int difficulty );
-    double GetGoldIncomeBonus( int difficulty );
+
+    // Returns an extra gold bonus modifier for AI based on difficulty level.
+    double GetGoldIncomeBonusForAI( const int difficulty );
```

Comment:
Since we are renaming this function we can name it from the lower character.
```suggestion
    double getGoldIncomeBonusForAI( const int difficulty );
```